apps/wallet-dashboard/lib/utils/vesting/vesting.spec.ts

```diff
@@ -278,16 +268,12 @@ describe('vesting overview', () => {
         const timelockObjects = mixedObjects.filter(isTimelockedObject);
         const availableClaiming = timelockObjects.reduce(
             (acc, current) =>
-                current.expirationTimestampMs <= Date.now()
-                    ? acc + bigIntRound(current.locked.value)
```

Comment:
I see that you are no longer using this here, but it is still being used other places of this file, should you also update those or no?